TorqueDataConnect/include/specials/TorqueDataConnectColumns.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * Special page that lists available spreadsheet columns
+ */
+
+class TorqueDataConnectColumns extends SpecialPage {
+  public function __construct() {
+    parent::__construct('TorqueDataConnectColumns');
+  }
+
+  public function execute($subPage) {
+    global $wgTorqueDataConnectGroup, $wgTorqueDataConnectWikiKey;
+    $this->setHeaders();
+    $ch = curl_init();
+    curl_setopt(
+      $ch,
+      CURLOPT_URL,
+      'http:/localhost:5000/api/sheets' .
```

Comment:
This isn't necessarily a comment about this commit, it's just a more general question, and it's as much for @frankduncan as it is for @YaxelPerez:

Is the "localhost:5000" convention for TorqueDataConnect something we've documented, and there's no reason for us to parameterize it?  If we have "http://localhost:5000/..." all over the code, that feels a bit weird to me, but maybe it's not weird as long as we've documented that that's always the address?  Still, I'd kind of prefer to see a variable like `$TorqueDataConnectPoint` or whatever -- if nothing else, it would prevent other reviewers from having the same question pop up for them.

Reply:
Nope!  It is the base address on which flask boots up, and never got around to parameterizing it.  Now is as good a time as any.